src/client/Microsoft.Identity.Client/PublicClientApplication.cs

```diff
@@ -271,5 +272,26 @@ public bool IsBrokerAvailable()
                 username,
                 password);
         }
+
+        /// <summary>
+        /// Used to determine if the currently available broker is able to perform Proof-of-Possesion.
+        /// </summary>
+        /// <returns>Boolean indicating Proof-of-Possesion is supported</returns>
+        /// <exception cref="MsalClientException"></exception>
+        public bool IsProofOfPosessionSupportedByClient()
+        {
+            if (DesktopOsHelper.IsWin10OrServerEquivalent())
```

Comment:
We should first check if the broker is enabled, then if the broker is InstalledAndInvokable and then if it supports POP. `IsWin10OrServerEquivalent` should not be used, it is an implementation detail of WAM's IsInstalledAndInvokable

Reply:
Blocking on this comment.